CDL-Dryad/dryad-app
# Auto-merge users with duplicate orcid/email

Closes https://github.com/CDL-Dryad/dryad-product-roadmap/issues/1923

To test: 
- In `stash_engine_users`, set two users to have the same ORCID.
- Login with that ORCID.
- The login should succeed, and the two user accounts should be merged into one.
